app/models/pusher.rb

```diff
@@ -124,7 +124,7 @@ def after_write
       Mailer.delay.gem_pushed(user.id, @version_id, notified_user.id)
     end
     Indexer.perform_later
-    rubygem.delay.reindex
+    ReindexRubygemJob.perform_later(rubygem:)
```

Comment:
1. I didn't know about this syntax, neat.
2. Isn't it easier to just take a single argument instead of a kwarg?

Reply:
Benefit to kwargs is that it's easier to make changes to the signature that are backwards compatible, which is needed because it's possible to deploy a version of the job that uses an old version of the args

Reply:
Oh, good point. Thanks!